tools/python/dump_subgraphs.py

```diff
@@ -47,7 +73,11 @@ def main():
         os.makedirs(out)
 
     model = onnx.load_model(model_path)
-    dump_subgraph(model, out)
+
+    # AttributeProto.type is available starting from IR_VERSION_2017_10_30
+    can_check_attribute_type = model.ir_version >= onnx.Version.IR_VERSION_2017_10_30
```

Comment:
this could be simplified if we only support models with this IR version (2) or later.